drivers/usb/usbstor/scsi.c

```diff
@@ -226,8 +226,6 @@ USBSTOR_CSWCompletionRoutine(
             CapacityData->LogicalBlockAddress = Response->LastLogicalBlockAddress;
             Irp->IoStatus.Information = sizeof(READ_CAPACITY_DATA);
        }
-
-       FreeItem(Context->TransferData);
```

Comment:
Where is this freed instead now?

Reply:
I spent some time trying understanding this :)
This pool was allocated in USBSTOR_SendCapacity (which I removed in this commit too). The purpose is to support returning both READ_CAPACITY_DATA and READ_CAPACITY_DATA_EX structures for SCSIOP_READ_CAPACITY.
But I see no READ_CAPACITY_DATA_EX usage in our codebase and looks like it is made for SCSIOP_READ_CAPACITY16 (64-bit version)

I'm going to drop that READ_CAPACITY_DATA_EX for now, what do you think?